gradle/spec/fixtures/buildfiles/root_build.gradle.kts

```diff
@@ -16,7 +16,7 @@ buildscript {
 
     repositories {
         jcenter()
-        maven("https://dl.bintray.com/magnusja/maven")
+        maven(url = "https://dl.bintray.com/magnusja/maven")
```

Comment:
Could we add both a repository _with_ and without the `url` just to make sure we don't break this in the future?